src/OrchardCore.Modules/OrchardCore.Forms/Entities/Scripting/ValidateRuleMethod.cs

```diff
@@ -0,0 +1,144 @@
+using System;

```

Comment:
Can you include a recipe (just attach under the PR) that contains a demonstration of how to use this, with a form?